Error: Navigational lists should be nested within a `nav` element
Error: Navigation, especially supplementary navigation, should have an accessible label
diff --git a/output/Additional Navigation Element/2/594caf29-05bb-4699-ad50-362347132ca9.html b/output/Additional Navigation Element/2/594caf29-05bb-4699-ad50-362347132ca9.html
index 9ac894b..3721a3b 100644
--- a/output/Additional Navigation Element/2/594caf29-05bb-4699-ad50-362347132ca9.html	
+++ b/output/Additional Navigation Element/2/594caf29-05bb-4699-ad50-362347132ca9.html	
@@ -1,5 +1,7 @@
-<ul>
-  <li><a href="/privacy">Privacy Policy</a></li>
-  <li><a href="/accessibility">Accessibility Statement</a></li>
-  <li><a href="/terms">Terms of Use</a></li>
-</ul>
\ No newline at end of file
+<nav aria-label="Policies and Legal Information">
+  <ul>
+    <li><a href="/privacy">Privacy Policy</a></li>
+    <li><a href="/accessibility">Accessibility Statement</a></li>
+    <li><a href="/terms">Terms of Use</a></li>
+  </ul>
+</nav>
\ No newline at end of file
